frog/imports/client/Wiki/index.js

```diff
@@ -52,13 +52,19 @@ export const LearningItem = dataFn.LearningItem;
 
 type WikiCompPropsT = {
   location: *,
-  match: {
+  match?: {
     params: {
       wikiId: string,
-      pageTitle: ?string
+      pageTitle: ?string,
+      instance?: string
     }
   },
-  history: Object
+  history: Object,
+  embedPage?: {
+    wikiId: string,
+    page: string,
```

Comment:
Could rename to pageTitle for more clarity.

Reply:
It is the pageId, I'll change the variable name and write comments

Reply:
Made the respective changes